docs/governance-strategy.md

```diff
@@ -0,0 +1,137 @@
+
+### Core team members
+
+Individuals who are particularly interested in the EBP community and have demonstrated a willingness to participate in our community and further its mission. They guide discussion, grow the community, contribute code, and generally help the project improve.
+
+Currently the core team is [defined here](https://executablebooks.org/en/latest/team.html).
+
+## Three sources of truth
+
+### `executablebooks/team-compass`: Team Policy
+
+Our Team Compass is the Source of Truth (SoT) for all organizational policy[^1]. The Team Compass should make it clear what is _policy_ and what is _guidance_. It should define clear sections that are community "musts" and others that are "guidelines and recommendations".
+
+It can also delegate SoT status to other places in the organization. It delegates the SoT for the MyST specification to `myst-spec/`.
+
+[^1]: A Team Compass is common in the Jupyter ecoystem (e.g., [here is JupyterHub's Team Compass](https://jupyterhub-team-compass.readthedocs.io/en/latest/index-team_guides.html). In fact, Jupyter's governance model will soon [require that sub-projects have their own Team Compass](https://jupyter.org/governance/software_subprojects.html?responsibilities-of-jupyter-subprojects)
+
+#### How to change team policy
+
+Take the following steps for changing any policy, strategy, or governance aspects of the Team Compass.
+
+1. **Open an issue** to explain the policy that you'd like to change, and why. Here is a rough template to follow:
+   ```
+   ## Context
+   
+   provide context needed to understand this proposal. Describe the problem this proposal will solve.
+   
+   ## Proposal
+   
+   describe your proposal in informal but specific terms.
+   
+   ## Impact
+   
+   describe the implications of this proposal and the impact it will have.
+   ```
+2. **Discuss and incorporate feedback** with others on the team. If there are objections or suggestions, do your best to incorporate them into your proposal.
+3. **Make a Pull Request** to the `Team Compass` repository (or another location if appropriate) and link it to the issue. This is the "formal change" that you wish to make.
+4. **Make a decision**. Steering Council members may approve PRs to change policy. To approve a change, use the "Approve" feature in the GitHub UI. To request blocking changes, use the "Request Changes" feature in the GitHub UI[^blocking]. PRs to change organizational policy may be merged when the following conditions are met:
+   - Have been open for five working days
+   - Have `Approval` from at least **One  Steering Council member**
+   - Have no `Request Changes` from a Steering Council member.
+   - If there are unresolveable objections from a Steering Council member, a decision to merge is made with a majority vote.
+   - If all steering council members `Approve`, it may be merged any time.
```

Comment:
*it may be merged any time.* -> *it may be merged any time, overriding the requirement of being open for five working days.*

(Just to be as clear as possible.)